src/LaunchDarkly.ServerSdk.Consul/ConsulDataStoreBuilder.cs

```diff
@@ -145,8 +146,12 @@ public IPersistentDataStoreAsync CreatePersistentDataStore(LdClientContext conte
                 client,
                 _existingClient != null,
                 _prefix,
-                context.Basic.Logger.SubLogger("DataStore.Consul")
+                context.Logger.SubLogger("DataStore.Consul")
                 );
         }
+
+        /// <inheritdoc/>
+        public LdValue DescribeConfiguration(LdClientContext context) =>
+            LdValue.Of("Consul");
```

Comment:
I forgot to call this out in the other PRs, but all three of these integrations were missing this implementation of the `IDiagnosticDescription` interface. This doesn't affect the database functionality, it just makes it so that when we collect metrics on how many people are using database integrations (assuming diagnostics are enabled), we will know if this one is being used.